dmscripts/generate_database_data.py

```diff
@@ -0,0 +1,48 @@
+import random
+import string
+import os
+from dmapiclient import DataAPIClient
+
+from typing import Optional
+
+USER_ROLES = [
+    'admin',
+    'admin-ccs-category',
+    'admin-ccs-sourcing',
+    'admin-manager',
+    'admin-framework-manager',
+    'admin-ccs-data-controller',
+    'buyer',
+    'supplier',
+]
+
+
+DEFAULT_PASSWORD = os.getenv("DM_DEFAULT_PASSWORD", default=None)
+
+
+def generate_email_address(username: Optional[int] = None) -> str:
+    """Generate an email of the form 123456@user.marketplace.team"""
```

Comment:
Why do we want emails of this form?

Reply:
That's one of the email domains that admin accounts are allowed to be on a non-production environment (see https://github.com/alphagov/digitalmarketplace-api/blob/main/config.py#L31) and also the one we already change user accounts to when we clean production data - https://github.com/alphagov/digitalmarketplace-aws/blob/main/scripts/clean-db-dump.sql#L6.

I'll add to this docstring to explain why